arm-postgresql/2017-04-30-preview/swagger/postgresql.json

```diff
@@ -39,11 +39,11 @@
         "tags": [
           "Servers"
         ],
-        "operationId": "Servers_CreateOrUpdate",
+        "operationId": "Servers_Create",
         "x-ms-examples": {
           "ServerCreate": { "$ref": "../examples/ServerCreate.json" }
         },
-        "description": "Creates a new server or updates an existing server. The update action will overwrite the existing server.",
+        "description": "Creates a new server, or will overwrite an existing server.",
```

Comment:
Good catch! 
@Suna-MS: could you please confirm this part ` or will overwrite an existing server` about this method ?